docs/docker_installation.rst

```diff
@@ -0,0 +1,74 @@
+.. _docker_image:
+
+Docker image
+============
+
+Get Docker
+----------
+
+The first step is to download and install Docker on your platform.
+Refer to the following Docker documentation and choose the best installation
+path for you: `Get Docker <https://docs.docker.com/get-docker/>`_
+
+Build the Image
+---------------
+
+VulnerableCode is distributed with ``Dockerfile`` and ``docker-compose.yml`` files
+required for the creation of the Docker image.
+
+Clone the git `VulnerableCode repo <https://github.com/nexB/vulnerablecode>`_,
+create an environment file, and build the Docker image:
+
+.. code-block:: bash
+
+    git clone https://github.com/nexB/vulnerablecode.git && cd vulnerablecode
+    make envfile
+    docker-compose build
+
+.. note::
+
+    The image will need to be re-built when the VulnerableCode app source code is
+    modified or updated via
+    ``docker-compose build --no-cache vulnerablecode``
+
+Run the Image
+-------------
+
+Run your image as a container
+
+.. code-block:: bash
+
+    docker-compose up
+
+
+At this point, the VulnerableCode app should be running at port ``8000`` on your Docker host.
+Go to http://localhost:8000/ on a web browser to access the web UI.
+Optionally, you can set ``NGINX_PORT`` environment variable in your shell or in the `.env` file to run on a different port than 8000.
+
+.. warning::
+
+    To access a dockerized VulnerableCode app from a remote location, the ``ALLOWED_HOSTS``
```

Comment:
We should be explicit that this could be security issue and there are several steps that may be needed to secure such a deployment. Alternative, we should not recommend this